.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,74 @@
+name: Deriv Lighthouse Audit
+
+on:
+  issue_comment:
+    types: [edited]
+
+jobs:
+  generate_lighthouse_audit:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Capture PR number
+        id: capture_pr_number
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+             core.setOutput("pr_number", context.issue.number)
```

Comment:
I believe you can skip this entire step and use `${{github.event.issue.number}}` to get the _"issue_number"_ which you can feed into `sticky-pull-request-comment` (all PRs are issues).